homework_4/bet_acceptance_api/internal/api/controllers/validators/betValidator.go

```diff
@@ -0,0 +1,29 @@
+package validators
+
+import "github.com/jelenanemcic/code-cadets-2021/homework_4/bet_acceptance_api/internal/api/controllers/models"
+
+const maxCoefficient = 10.0
+const minPayment = 2.0
+const maxPayment = 100.0
```

Comment:
This should be part of the configuration, not hardcoded.

What if there's multiple countries with different laws where I want to deploy this same service but with these attributes being different?